src/front-door/azext_front_door/custom.py

```diff
@@ -285,7 +285,7 @@ def configure_fd_frontend_endpoint_disable_https(cmd, resource_group_name, front
 
 def configure_fd_frontend_endpoint_enable_https(cmd, resource_group_name, front_door_name, item_name,
                                                 secret_name=None, secret_version=None,
-                                                certificate_source=None, vault_id=None):
+                                                certificate_source='FrontDoor', vault_id=None):
```

Comment:
Is this a breaking change?

Reply:
Not really, previously the default value was None and command was not doing anything(Ideally if should enable https on custom domain). This change would fix that.

Reply:
@tjprescott